frontend/src/app/supportAdmin/pageTitles.ts

```diff
@@ -0,0 +1,11 @@
+// js file so imports to cypress work
+export const addNewDevicePageTitle = "Add a new device";
```

Comment:
looks like based on Laura's comments, this was suppose to be "Add new device" without the "a"

Reply:
thanks for the catch elisa!